### PR TITLE
Improve log after SECURITY-626

### PR DESCRIPTION
**Pull request by Wadeck**
No Jira ticket.

Provide better log information for admin still trying to use user/password + crumb in script while the API Token is easier to use since ~1 year.

Now the crumb is not usable to its binding with the session.

The redirect URL requires to be created and linked to https://jenkins.io/doc/upgrade-guide/2.176/#SECURITY-626.
=> Companion PR: https://github.com/jenkins-infra/jenkins.io/pull/2785.

### Proposed changelog entries

* (only for admin) Improve log message in case a crumb was likely used in a script

### Submitter checklist

- [n/a] JIRA issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [n/a] Appropriate autotests or explanation to why this change has no tests
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a JIRA issue should exist and be labeled as `lts-candidate`

### CODE REVIEW

**Review by varyvol — APPROVED** [review]

**Comment by daniel-beck** [comment]
> The redirect URL requires to be created and linked to https://jenkins.io/doc/upgrade-guide/2.176/#SECURITY-626.

Please file a companion PR in https://github.com/jenkins-infra/jenkins.io/

**Review by daniel-beck — APPROVED** [review]

**Review by jvz — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
If @daniel-beck is fine with naming, it is ready to go

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
labelled as rfe, unless anyone knows of a better category

**Review by res0nance — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
I plan to merge it tomorrow if no negative feedback. Or somebody else will merge it.
@res0nance FYI https://groups.google.com/forum/#!topic/jenkinsci-dev/1ioTdU4784M

**Comment by oleg-nenashev** [comment]
Not sure about skipping the changelog, but I leave it to the security team

**Pull request merged**
